man/man5/zfs-module-parameters.5

```diff
@@ -3656,6 +3656,27 @@ Default value: \fB0\fR.
 
 .sp
 .ne 2
+.na
+\fBzfs_zevent_retain_max\fR (int)
+.ad
+.RS 12n
+Maximum recent zevent records to retain for duplicate checking.  Setting
+this value to zero disables duplicate detection.
+.sp
+Default value: \fB3000\fR.
```

Comment:
This doesn't agree with the source code which defaults to 2000.

Reply:
good catch! fixed